Logos/Forcepoint.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please follow logo guidelines in step 5 of https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors#build-the-connector as there are changes needed for this logo to be deployed fine. Pasting here:
The company logo adhering to the following requirements in the 'Logo' folder
Logo needs to be in SVG format and under 5 Kb
Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
Do not use xlink:href - use inline instead
Do not use title tag
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
Logo scales well to fit in a 75 px square
SVG code file is formatted well for readability